backend/api/MQTT/MessageModels/IsarMission.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Text.Json.Serialization;
```

Comment:
Should the IsarMission topics be defined in Isar and not "duplicated" in Flotilla?

Reply:
Yes, it would be great if there was a way to somehow "import" these models, I made an [issue](#211 ) on this for future exploration.